tetracionist/sqlfluff-action
# Releases/v0.1.0 (#4)

* * Updated project metadata
* Added uv
* Added sqlfluff

* remove wait.ts

* update code

* try again

* added actions core

* compile it properly

* add pyproject.toml to dist

* Update pyproject.toml

* project-safe name

* if dbt

* Try again

* try running dbt deps

* use the right target

* print password debug

* recompile

* Use relative paths

* test with binary

* add missing uv env

* add dbt binary

* Make sure we get indirect deps too

* attempt to fix git diff

* Use the base ref instead

* Add baseref

* use origin

* Add reviewdog

* add path

* try again

* install pipx globally

* look at the binary folder

* add await

* testing reviewdog part

* try again

* Try without error

* parse json file

* catch sqlfluff error

* try to get reviewdog to do something

* correct newline character

* use file filter mode

* Fail on error

* mvp
